spec/spec_helper.rb

```diff
@@ -1,14 +1,9 @@
-require 'inherited_class_var'
-require 'bundler/setup'
-require 'coveralls'
-
-begin
-  require 'pry'
-rescue LoadError
-end
-
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```

Comment:
https://gist.github.com/lroggendorff/3908654 -- default when you make a gem via `bundler`